app/components/treatment/Treatment.js

```diff
@@ -98,7 +106,7 @@ export class Treatment extends Component {
 
 // -=-=-=-=-=-= CONTAINER =-=-=-=-=-=-
 
-const mapState = ({ user, treatment }) => ({ user, treatment })
+const mapState = ({ user, plan }) => ({ user, plan })
```

Comment:
This will work, but I think it would be easier to write a selector here and only pass down the treatment you need: `treatment: plan.treatments.find(treatment => treatment.id === currentTreatmentId)` where `currentTreatmentId` is also on the state